src/frontends/tests/frontend/shared/src/op_extension.cpp

```diff
@@ -31,13 +31,13 @@ void FrontEndOpExtensionTest::initParamTest() {
 
 ///////////////////////////////////////////////////////////////////
 
-TEST_P(FrontEndOpExtensionTest, TestOpExtensionVec) {
+TEST_P(FrontEndOpExtensionTest, DISABLED_TestOpExtensionVec) {
```

Comment:
Can we create a ticket to enable these tests?

Reply:
Ticket created: 83471